### PR TITLE
Update TestResources to 2.0.16

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@jaredpar, @AlekseyTs, please review.

cc @dotnet/roslyn-compiler

**Comment by AlekseyTs** [comment]
>         NetStandard30,

We probably should rename this to NetCoreApp30

---
Refers to: src/Test/Utilities/Portable/TargetFrameworkUtil.cs:21 in 7189736. [](commit_id = 7189736ae605f6aca020c7f294e76fe47757c8e5, deletion_comment = False)

File: src/Test/Utilities/Portable/TargetFrameworkUtil.cs [hunk 1]
```diff
@@ -81,7 +81,7 @@ public static class TargetFrameworkUtil
...
         public static ImmutableArray<MetadataReference> Mscorlib461ExtendedReferences => ImmutableArray.Create<MetadataReference>(Net461.mscorlibRef, Net461.SystemRef, Net461.SystemCoreRef, Net461.SystemValueTupleRef, Net461.SystemRuntimeRef, Net461.netstandardRef);
         public static ImmutableArray<MetadataReference> NetStandard20References => ImmutableArray.Create<MetadataReference>(NetStandard20.NetStandard, NetStandard20.MscorlibRef, NetStandard20.SystemRuntimeRef, NetStandard20.SystemCoreRef, NetStandard20.SystemDynamicRuntimeRef);
-        public static ImmutableArray<MetadataReference> NetStandard30References => ImmutableArray.Create<MetadataReference>(NetStandard30.NetStandard, NetStandard30.MscorlibRef, NetStandard30.SystemRuntimeRef, NetStandard30.SystemCoreRef, NetStandard30.SystemDynamicRuntimeRef, NetStandard30.SystemConsoleRef);
+        public static ImmutableArray<MetadataReference> NetStandard30References => ImmutableArray.Create<MetadataReference>(NetCoreApp30.NetStandard, NetCoreApp30.MscorlibRef, NetCoreApp30.SystemRuntimeRef, NetCoreApp30.SystemCoreRef, NetCoreApp30.SystemDynamicRuntimeRef, NetCoreApp30.SystemConsoleRef);
```
Comment (AlekseyTs) [inline src/Test/Utilities/Portable/TargetFrameworkUtil.cs]: >NetStandard30References [](start = 56, length = 23)

We probably should rename this to NetCoreApp30References

**Comment by AlekseyTs** [comment]
It looks like CI jobs are failing. Did we publish a new Nuget package for the resources?

**Comment by AlekseyTs** [comment]
Done with review pass (iteration 1)

**Pull request merged**